src/mjolnir/adminbuilder.cc

```diff
@@ -12,206 +12,169 @@
 
 #include "config.h"
 
-/* Need to know which geos version we have to work out which headers to include */
```

Comment:
dont look at the diff, just do view file. the sqlite stuff is unchanged but the rest i rewrote (though i used the same basic structure)